src/app/pages/OpenWalletPage/index.tsx

```diff
@@ -65,6 +65,23 @@ export function SelectOpenMethod() {
           />
         )}
       </Box>
+
```

Comment:
(I would slightly prefer it on https://wallet.oasisprotocol.org/open-wallet/mnemonic page, instead of https://wallet.oasisprotocol.org/open-wallet)

Reply:
Let's leave it here for now and if we have so many warnings that the open-wallet page will become too cluttered, we'll move them to specific subpages.